workspace/notebook/py_green_specialist_case_standardised_post_validation_check.json

```diff
@@ -0,0 +1,606 @@
+					"#ignore FutureWarning messages \n",
+					"import warnings\n",
+					"warnings.filterwarnings(\"ignore\", message=\"iteritems is deprecated\")\n",
+					""
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"##### Get Storage account"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"#Get Storage account name\n",
+					"storage_account=mssparkutils.notebook.run('/utils/py_utils_get_storage_account')\n",
+					"print(storage_account)"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"##### Enable message logging"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"%run utils/py_logging_decorator"
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"##### Define all storage paths"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"# Define delta table names, database name and table/file paths\n",
+					"source_database_name = \"odw_standardised_db\"\n",
+					"\n",
+					"source_delta_table = f\"{source_database_name}.green_specialist_case\"\n",
+					"csv_file_path = f\"abfss://odw-raw@{storage_account}Green_Specialist_Case\"\n",
+					"\n",
+					"# Date and csv file configuration\n",
+					"current_date = datetime.now().strftime(\"%Y-%m-%d\")\n",
+					"timestamp = datetime.now().strftime(\"%Y%m%d_%H%M\")\n",
+					"csv_filename = f\"rejected_green_case_records_{timestamp}.csv\"\n",
+					"\n",
+					"storage_name = storage_account.replace(\".dfs.core.windows.net/\", \"\")\n",
+					"start_time = datetime.now()\n",
+					"truncated_error = \"\"\n",
+					""
+				],
+				"execution_count": null
+			},
+			{
+				"cell_type": "markdown",
+				"metadata": {
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
+				"source": [
+					"##### Define Process Delta table update function"
+				]
+			},
+			{
+				"cell_type": "code",
+				"source": [
+					"# Get latest folder\n",
+					"def get_latest_folder(path):\n",
+					"    folders = [f.name for f in mssparkutils.fs.ls(path) if f.isDir]\n",
+					"    folders = sorted([f for f in folders if re.match(r\"\\d{4}-\\d{2}-\\d{2}\", f)], reverse=True)\n",
+					"    return folders[0] if folders else None\n",
+					"\n",
+					"# Date columns to validate\n",
+					"DATE_COLUMNS = [\n",
+					"    \"ReceiptDate\", \"ValidDate\", \"StartDate\", \"QuDate\", \"QuRecDate\",\n",
+					"    \"6Weeks\", \"8Weeks\", \"9Weeks\", \"EventDate\", \"DecisionDate\",\n",
+					"    \"DateWithdrawnorTurnedAway\"\n",
```

Comment:
Imo this doesn't scale well. Having to maintain a list of cols here is quite a long process. Would be good to have this be configurable from the calling notebook